config/develop/glue-jobs.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
 parameters:
   SourceBucketName: {{ stack_group_config.artifact_bucket_name }}
   JobRole: !stack_output_external glue-job-role::RoleArn
-  S3OutputBucketName: !stack_output_external bridge-downstream-dev-parquet-bucket::BucketName
+  S3BucketName: !stack_output_external bridge-downstream-dev-intermediate-bucket::BucketName
```

Comment:
The name was changed because S3OutputBucketName is misleading -- this value is not actually what's being used to write the output. That's passed (confusingly) at the workflows level instead. But we pass in something here to write temp files too, as well as logs. Would be good eventually to be more consistent about this.